# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1260254695 18000
#      Tue Dec 08 01:44:55 2009 -0500
# Node ID 1789861b8ef8601a74aa24fc1cb06a873cb6af00
# Parent  85389b4e3491a5937453530fb9ad5eab9c61ac59
scripts/Makeilfe.am (DOCSTRINGS): don't use $(simple_move_if_change_rule)

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2009-12-08  John W. Eaton  <jwe@octave.org>
+
+	* Makefile.am (DOCSTRINGS): Don't use $(simple_move_if_change_rule).
+
 2009-12-07  Jaroslav Hajek  <highegg@gmail.com>
 
 	* general/interp1.m: Allow discontinuities (jumps) for the "nearest" and
 	"linear" methods. Document the feature and add a few tests.
 
 2009-12-06  Rik <rik@nomad.inbox5.com>
 
 	* Makefile.am: Distribute DOCSTRINGS so that documentation will not require
diff --git a/scripts/Makefile.am b/scripts/Makefile.am
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -323,17 +323,17 @@ EXTRA_DIST += \
   $(IMAGES) \
   $(FCN_FILES) \
   $(FCN_FILES_IN)
 
 all-local: $(GEN_FCN_FILES) $(PKG_ADD_FILES) DOCSTRINGS
 
 DOCSTRINGS: mkdoc $(FCN_FILES) $(FCN_FILES_IN) gethelp$(BUILD_EXEEXT) 
 	$(srcdir)/mkdoc "$(srcdir)" $(FCN_FILES) $(FCN_FILES_IN) > $@-t
-	$(simple_move_if_change_rule)
+	mv $@-t $@
 
 gethelp$(BUILD_EXEEXT): gethelp.cc
 	$(BUILD_CXX) $(BUILD_CXXFLAGS) -o $@ $^ $(BUILD_LDFLAGS)
 
 $(GEN_FCN_FILES) : %.m : %.in Makefile
 	$(do_subst_config_vals)
 
 check-m-sources:
